src/dbcp/transform/lbnl_iso_queue.py

```diff
@@ -154,6 +167,7 @@ def _clean_all_iso_projects(projects: pd.DataFrame) -> pd.DataFrame:
             "raw_state_name",
             "utility_clean",  # derived in _prep_for_deduplication
             "resource_type_1",
+            "queue_status",
```

Comment:
I added `queue_status` to the deduplication key now that we have operational and withdrawn projects.